### ISSUE TITLE
Trouble Extracting Parasaur and Jerboa on Scorched Earth

### ISSUE THREAD

**Issue by RodanThrelos** [cadon/ARKStatsExtractor#300]
As the title says, I keep running into issues when attempting to extract the values on my Parasaur and Jerboa on my family's Scorched Earth server.  I was able to (finally) extract my Pulminoscorpius, but I can't get my Parasaur and Jerboa to work.  Attached are my stats, as well as my server settings.

I am running the latest version of the program, v0.23.2 with values from 257.47.0.

![ark breeding problem](https://cloud.githubusercontent.com/assets/28611435/26063093/ec2bcba4-395a-11e7-838f-e6c26c2434ff.jpg)
![ark breeding problem 3](https://cloud.githubusercontent.com/assets/28611435/26063095/ee422fa0-395a-11e7-8418-07d70a46fff7.jpg)
![ark breeding problem 2](https://cloud.githubusercontent.com/assets/28611435/26063100/f07f7ce6-395a-11e7-9cf2-9a5e4046741d.png)

**Comment by cadon** [comment]
Hi, these creatures were probably spawned in by an admin. They have wild-levels that are not possible when they spawn naturally. In the settings you have the max-wild-level set to 120, which means, wild creatures have levels of multples of 4 (4,8,12,...,116,120). The Parasaur has a wild level of 3, the Jerboa of 1, so the valid possibilities are sorted out by the tool. To disable the sorting out of impossible levels (because apparently they are not impossible), you can disable the setting "Consider Wild-Level-Steps" in the general settings (bottom left). This will make it possible to extract these creatures. The only downside is, for heavily leveled creatures there might be more possiblities to choose from (if a stat is yellow), but usually it's not a big deal.

**Comment by RodanThrelos** [comment]
Cadon,

Thank you for your quick reply.  Not sure what happened with those creatures; I was the only person to play on the server, and I definitely didn't spawn them in.  Anyway, I have another problem with a recently-tamed Pulminoscorpius on my TheIsland server.  The settings are the same as before, and I've quadruple-checked them.  App is having trouble extracting the health.

![ark breeding problem 4](https://cloud.githubusercontent.com/assets/28611435/26130467/dc8155d0-3a62-11e7-852a-9acbce097cfb.jpg)

**Comment by cadon** [comment]
Maybe the difficulty setting was changed, so they spawned in when their level was possible? Or there's another mechanic I haven't considered yet.

Sorry about the scorpions. Long story short: just leave them out. They have a strange behaviour regarding their health-stat. It's in some kind of interstate between their release value and a later (over a year ago) applied nerf / adjustment to the health stat. It doesn't make sense. I decided to not spend more time in trying to understand how their stat is calculated, it's changing (sometimes the old values which shouldn't exist anymore, sometimes the new ones, or a mix).
For more infos about the scorpion issue, see #16, #60, #92, #162, #207.
